Library/Homebrew/software_spec.rb

```diff
@@ -107,6 +108,10 @@ def resource(name, klass = Resource, &block)
     end
   end
 
+  def get_resource
+    @resource
+  end
```

Comment:
I've grepped where I use `get_resource`, it seems I don't need it at all anymore. Maybe at the first place there was more sense in it, but now I don't need to get `resource`, all I need is `downloader` and `version` and I can get them using delegators already defined in `SoftwareSpec`.

```ruby
  def_delegators :@resource, :version, :mirror, *Checksum::TYPES
  def_delegators :@resource, :downloader
```